ntoskrnl/io/pnpmgr/pnpinit.c

```diff
@@ -63,7 +63,7 @@ PiInitCacheGroupInformation(VOID)
 
         /* Bogus data */
         PiInitGroupOrderTableCount = 0;
-        PiInitGroupOrderTable = (PVOID)0xBABEB00B;
+        PiInitGroupOrderTable = (PVOID)(ULONG_PTR)0xBABEB00BBABEB00BULL;
```

Comment:
Yeah xD

Reply:
Yeah, at some point we should change these things to be more neutral. I understand not doing that as part of this though.